tests/Validator/HostnameTest.php

```diff
@@ -60,6 +60,20 @@ public function testIsValid()
         $this->assertEquals(false, $validator->isValid('myweb.com/blog'));
         $this->assertEquals(false, $validator->isValid('myweb.com/blog/article1'));
 
+        // Max length test
+        $domain = '';
+        $chunk = 'bestdomain'; // 10 chars
+        for($i = 0; $i < 25; $i++) {
+            $domain .= $chunk; // 250 total
+        }
```

Comment:
I think https://www.php.net/manual/en/function.str-repeat.php this is what you are looking for 😉

Reply:
Woo cool 👀 Applied